notebooks/rfi_inspect.ipynb

```diff
@@ -26,7 +26,9 @@
     "from astropy import units\n",
```

Comment:
<p>Is this still supposed to be here?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HERA-Team/hera_notebook_templates/pull/5/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>Yes I believe that's the XRFI detection significance plotting code. I've recaptioned the figures so that they are consistently captioned below where they appear. It was a little inconsistent which may have been confusing.</p>